api/firmware/device.go

```diff
@@ -114,15 +114,28 @@ type DeviceInfo struct {
 	SecurechipModel string `json:"securechipModel"`
 }
 
+// DeviceInfoREQ_INFO is the data returned from the REQ_INFO api call.
+type DeviceInfoREQ_INFO struct {
```

Comment:
can unexport the struct, and i'd simply name `info struct` or sth, `REQ_INFO` is not really following any naming pattern in Go :P